File: LeetCode/JavaScript/#217 Contains Duplicate.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
var containsDuplicate = function(nums) {
    var res = Array.from(new Set(nums));
    return !(res.length===nums.length);
};

var containsDuplicate = function(nums) {
    var len = nums.length;
    var res = {};
    for(let i= 0;i<len;i++){
    	if(res[nums[i]]){
    		return true;
    	}else{
    		res[nums[i]] = 1;
    	}
    }
    return false;
};

File: LeetCode/Python/#217 Contains Duplicate.py
# class Solution(object):
#     def containsDuplicate(self, nums):
#         """
#         :type nums: List[int]
#         :rtype: bool
#         """
#         for i in set(nums):
#             if nums.count(i)>1:
#                 return True
#         return False
#
#


class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        return len(list(set(nums)))!=len(nums)

